packages/vm/tests/runner.ts

```diff
@@ -0,0 +1,63 @@
+const level = require('level')
+import { Transaction } from '@ethereumjs/tx'
+import VM from '../lib'
+import { SecureTrie as Trie } from '@ethereumjs/trie'
+import { DefaultStateManager } from '../lib/state'
+import Common from '@ethereumjs/common'
+import { Block } from '@ethereumjs/block'
+
+const main = async () => {
+  const txData = {
+    nonce: '0x1a',
+    gasPrice: '0xba43b7400',
+    gasLimit: '0x493e0',
+    to: '0x9ca228250f9d8f86c23690074c2b96d5f5479f79',
+    value: '0x0',
+    data:
+      '0xa9059cbb000000000000000000000000bfe465e7eb5a2928b5bf22bef93ad06089dc6179000000000000000000000000000000000000000000000000006a94d74f430000',
+    v: '0x1c',
+    r: '0xc04f048766bea3b20dcea6d2fcaecc92f1a615d337606ab6eec2a1e8e1f27bfe',
+    s: '0x1c57e62653724560b06c95be750d958840db9fa44ffa777b6625b344bfadbbcf',
+  }
+  const common = new Common({ chain: 'mainnet', hardfork: 'chainstart' })
+  const tx = Transaction.fromTxData(txData, { common })
+  const trie = new Trie(level('/Users/jochem/Library/Ethereum/ethereumjs/mainnet/state/'))
+
+  // Ensure we run on the right root
+  trie._setRoot(
+    Buffer.from('9dceb42e227ee6a244449d1d92cc5e0c17c63b520d0ff050d943baa8055d0ca6', 'hex')
+  )
+
+  const stateManager = new DefaultStateManager({ trie, common })
+  const vm = new VM({ stateManager, common })
+
+  vm.on('step', (o: any) => {
+    if (o.opcode.name == 'SSTORE') {
+      console.log(
+        o.stack[o.stack.length - 1].toString('hex'),
+        o.stack[o.stack.length - 2].toString('hex')
```

Comment:
Logs the slot and the value in case we SSTORE. 

In the 0x83 contract, we put 0x00..3c in slot 0x00..02. This [should be](https://etherscan.io/tx/0x8420dbe268209c4e921bf6b78bc9fb959d9d9553fd16aacb9dd8cad955dbe729#statechange) `0x0000000000000000000000000000000000000000000000000000000055f4f99f`. 

I explicilty checked on my full node that this storage actually does exist and is not wiped if the code deposit goes out of gas. So, it should be the value above.

Reply:
Ah, I wondered during the night if would need/want some per-opcode debugging functionality, this gives me some confirmation 😋 , will see if I can add to #1080.

The we can do something like `DEBUG=vm:opcodes:sstore npm run test.ts` out of the box. 😄  (maybe this naming is a bit long, will think about that)

Reply:
Some other question here out of interest: how do you check this on your full node? 🤔

Reply:
In a terminal, `geth attach` to the current datadir (mainnet). This gives access to web3 methods, especially `eth` ones which are interesting. 

If you call `eth.getStorageAt(address, slot)` you get the storage at the slot. Note: this checks by default on the latest block, but since we `CREATE`d the 0x83 address, it is not possible to change the code on this address. Since there is no code deposited, this means that the storage cannot be changed. Hence, by this logic, if we query `eth.getStorageAt("0x8398ff6c618e9515468c1c4b198d53666cbe8462", "0x0000000000000000000000000000000000000000000000000000000000000000")`, we get the storage value which was deposited in the transaction 0x8420dbe268209c4e921bf6b78bc9fb959d9d9553fd16aacb9dd8cad955dbe729. 

If there would be code, then we would not be sure if that value was actually deposited in the transaction (it could have changed. Then we would either query `eth.getStorageAt("0x8398ff6c618e9515468c1c4b198d53666cbe8462", "0x0000000000000000000000000000000000000000000000000000000000000000", 226522)` (this would be the storage value of the slot *after* running each transaction of the block (so after our consensus failing transation, i.e. the deposited storage in most cases if no other transaction in the block interacted with the contract), or we would query `eth.getStorageAt("0x8398ff6c618e9515468c1c4b198d53666cbe8462", "0x0000000000000000000000000000000000000000000000000000000000000000", 226521)`, which would be the value after we had processed the earlier block, i.e. the value in the slot right before we start processing the transaction. 

Since these blocks are very old, this would usually require that we have an archive node (which I don't have).

Reply:
That's super interesting, thanks for the extensive write-up! 😄 

Just discovered that `Ethers` has similar functionality with [getStorageAt()](https://docs.ethers.io/v5/single-page/#/v5/api/providers/provider/-%23-Provider-getStorageAt), as a side note.

Reply:
(getting more and more of an `Ethers` user finally 😅 )